site/docs/flink.md

```diff
@@ -522,6 +522,18 @@ FlinkSink.forRowData(input)
 env.execute("Test Iceberg DataStream");
 ```
 
+## Streaming upsert write
+
+Iceberg supports upsert based on the primary key when writing streaming data.
```

Comment:
Would it be helpful to state here that this requires v2 tables?

Reply:
Thank you for your reply, this can be removed. The introduction of this feature is needed for new users of iceberg

Reply:
yeap,  update